toptal/picasso
# fix(tooltip): trigger immediately on touch

[TEA-1673](https://toptal-core.atlassian.net/browse/TEA-1673)

### Description

Restores the intended behavior introduced on #1190.

[Material-UI changed the way `disableTouchListener` works](https://github.com/mui-org/material-ui/commit/5627473d6cc24be4ff750be239892f835701b2e0#diff-db872c13568b6f9d9dd4ea1f5e876cdb) (we were basically relying on a quirk of the component) and we didn't change it on our side.

Also, someone didn't commit the changes on `yarn.lock` so `yarn install` yielded some changes.

### How to test

Open the Tooltip trigger example on a small screen and touch on the "Hover" button. The content should appear.

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
